CHANGELOG.md

```diff
@@ -3,14 +3,23 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v2.0.0dev - [date]
+## 2.0.0 Silver Zebra - [2021-05-12]
```

Comment:
As a naming scheme I was thinking of `metal`+`animal classically kept in cages`. I am not sure zebra fall into that category 🤔

Reply:
Aaaah good idea :rofl: 
You choose one then :)

Reply:
The codename generator oracle has spoken 😀:
```suggestion
## 2.0.0 Nickel Chicken - [2021-05-12]
```

Reply:
Nice :grin: